local-volume/README.md

```diff
@@ -37,11 +37,19 @@ What doesn't work and workarounds:
         * Give your pods high priority.
         * Run a workaround controller that unbinds PVCs for pods that are
           stuck pending. TODO: add link
-* External provisioner cannot correctly detect capacity of mounts added after it
-  has been started.
-    * This requires mount propagation to work, which is targeted for 1.9.
-    * Workaround: Before adding any new mount points, stop the daemonset, add
-      the new mount points, start the daemonset.
+* The provisioner does not correctly detect mounts added after it
+  has been started, and it incorrectly creates local PVs with
+  the root filesystem capacity. This situation consistently manifests due to
+  the non-atomic nature of discovery of the directory for the new mountpoint
```

Comment:
It's because mount propagation is not enabled, so the provisioner cannot see new mount points.

Reply:
Thanks for the fast feedback. I've updated the description accordingly.